ext/winevt/winevt_c.h

```diff
@@ -47,6 +53,8 @@ extern VALUE rb_cBookmark;
 extern VALUE rb_cSubscribe;
 extern VALUE rb_eWinevtQueryError;
 
+extern const LocaleInfo default_locale;
```

Comment:
Using a pointer is better to avoid copying whole struct data.

```diff
- extern const LocaleInfo default_locale;
+ extern const LocaleInfo *default_locale;
```

Reply:
Sorry, it's misleading.
`default_locale` shouldn't be a pointer while `localeInfo` in `WinevtQuery` should be a pointer.